### DISCUSSION TITLE
21.02如何编译中文版？

### DISCUSSION THREAD

**Discussion by Thanf-code** [immortalwrt/immortalwrt#483] [Q&A] [answered]

**Comment by 1715173329** [comment, marked as answer]
我想你在 menuconfig -> LuCI 目录里面能找到答案。

**Reply by Thanf-code** [reply]
> 我想你在 menuconfig -> LuCI 目录里面能找到答案。

1.在config里添加CONFIG_LUCI_LANG_zh_Hans=y
编译成功
但没有生成sysupgrade.bin文件
且生成的文件比较大
2.只能ssh进去modules->translation->simplified chinese  试试看

**Reply by Thanf-code** [reply]
即使第二种方法（menuconfig ->luci->modules->translation->simplified zh_Hans） 编译出来也是跟第一种方法（.在config里添加CONFIG_LUCI_LANG_zh_Hans=y）是一样的

**Comment by i4210577** [comment]
在config里添加CONFIG_LUCI_LANG_zh_Hans=y，CONFIG_LUCI_LANG_zh_cn=y
编译成功没中文版
源码https://github.com/1715173329/immortalwrt
分支master-ax3000
处理器型号ipq807x

**Reply by chobitsos** [reply]
> 在config里添加CONFIG_LUCI_LANG_zh_Hans=y，CONFIG_LUCI_LANG_zh_cn=y 编译成功没中文版 源码https://github.com/1715173329/immortalwrt 分支master-ax3000 处理器型号ipq807x

你要在make menuconfig里面选menuconfig ->luci->modules->default-settings-chn
你直接修改config文件没有意义

**Comment by 1715173329** [comment]
勾选 default-settings-chn。
